com/uber/kafkaSpraynozzle/KafkaSpraynozzle.java

```diff
@@ -81,10 +87,16 @@ public static void main(String[] args) {
         ConsumerConfig consumerConfig = new ConsumerConfig(kafkaProps);
         ConsumerConnector consumerConnector = Consumer.createJavaConsumerConnector(consumerConfig);
         HashMap<String, Integer> topicParallelism = new HashMap<String, Integer>();
-        topicParallelism.put(topic, partitionCount);
+        for (int i = 0; i < topics.length; i++) {
+            topicParallelism.put(topics[i], partitionCount);
+        }
         Map<String, List<KafkaStream<Message>>> topicMessageStreams = consumerConnector.createMessageStreams(topicParallelism);
-        List<KafkaStream<Message>> streams = topicMessageStreams.get(topic);
-        ExecutorService executor = Executors.newFixedThreadPool(threadCount+partitionCount+1);
+        ArrayList<List<KafkaStream<Message>>> streams = new ArrayList<List<KafkaStream<Message>>>();
+        for (int i = 0; i < topics.length; i++) {
+            List<KafkaStream<Message>> stream = topicMessageStreams.get(topics[i]);
+            streams.add(stream);
+        }
+        ExecutorService executor = Executors.newFixedThreadPool(threadCount+(partitionCount*topics.length)+1);
```

Comment:
Adding spaces, but the `javac` linter isn't warning about that to me? What lint settings are you using?